tool/tbot/config/configtemplate.go

```diff
@@ -91,11 +143,31 @@ func (c *TemplateConfig) UnmarshalYAML(node *yaml.Node) error {
 }
 
 func (c *TemplateConfig) CheckAndSetDefaults() error {
+	templates := []interface{ CheckAndSetDefaults() error }{
+		c.SSHClient,
+		c.Identity,
+		c.TLS,
+		c.TLSCAs,
+		c.Mongo,
+		c.Cockroach,
+	}
+
 	notNilCount := 0
+	for _, template := range templates {
+		// Note: this check is fragile and will fail if the templates aren't
+		// all simple pointer types. They are, though, and "correct" solution
+		// is insane, so we'll stick with this.
+		if template == nil || reflect.ValueOf(template).IsNil() {
```

Comment:
`template == nil` is always false here, right? they're all non-nil `interface { CheckAndSetDefaults() error }`s (which might be nil pointers internally).

Reply:
Yep, will remove.